cmd/kapacitord/run/command.go

```diff
@@ -127,6 +127,11 @@ func (cmd *Command) Run(args ...string) error {
 	if err := s.Open(); err != nil {
 		return fmt.Errorf("open server: %s", err)
 	}
+
+	if err := s.LoadService.Load(); err != nil {
```

Comment:
Why isn't this part of the Open process for the Server?

Reply:
fixed.